### ISSUE TITLE
Validation of valueBoolean parameters has false negatives

### ISSUE THREAD

**Issue by yunosh** [dcarbone/php-fhir#136]
This unit test incorrectly fails with the validation error {"errors":{"parameter.0":{"valueBoolean":{"value":{"value":{"pattern":"Field \"value\" on type \"boolean-primitive\" value of \"1\" does not match pattern: /^true|false$/"}}}}}}

```php
<?php

namespace App\Tests;

use DCarbone\PHPFHIRGenerated\R4\PHPFHIRResponseParser;
use PHPUnit\Framework\TestCase;

class BooleanValidationTest extends TestCase
{
    public function testBooleanValidation(): void
    {
        $fhir = <<<FHIR
<?xml version="1.0" encoding="utf-8"?>
<Parameters xmlns="http://hl7.org/fhir">
	<meta>
		<profile value="https://e-lb.de/fhir/StructureDefinition/KK_ELB_ParticipationStatusResponseParameters"/>
	</meta>
	<parameter>
		<name value="isParticipating"/>
		<valueBoolean value="true"/>
	</parameter>
</Parameters>
FHIR;

        $parser       = new PHPFHIRResponseParser();
        $fhirResponse = $parser->parseXml($fhir);
        $validationErrors = $fhirResponse->_getValidationErrors();
        self::assertEquals([], $validationErrors);
    }
}
```

**Comment by dcarbone** [comment]
@yunosh: I've released v3.1.0 with numerous fixes around validation, please let me know if you encounter any other issues!

**Comment by yunosh** [comment]
Works fine for me now, thanks!
